iceoryx_examples/waitset_in_c/ice_c_waitset_timer_driven_execution.c

```diff
@@ -72,7 +73,8 @@ void* cyclicTriggerCallback(void* dontCare)
 bool createThread(pthread_t* threadHandle, void* (*callback)(void*))
```

Comment:
Oh, I guess I forgot to change this to `int` after noticing that pthread_create also returns an int. Could you please change the return type?